src/dev/license_checker/config.ts

```diff
@@ -69,6 +69,7 @@ export const LICENSE_ALLOWED = [
   'Python-2.0',
   '(Apache-2.0 AND MIT)',
   'BlueOak-1.0.0',
+  '(MPL-2.0 OR Apache-2.0)',
```

Comment:
Interesting that this is necessary, shouldn't it be covered by this block where `DEV_ONLY_LICENSE_ALLOWED` is concatenated to `LICENSE_ALLOWED` (since `'MPL-2.0'` is in DEV_ONLY... below on line `77`)?

https://github.com/elastic/kibana/blob/main/src/dev/license_checker/run_check_licenses_cli.ts#L38-L43

Even though this list says it's deps `OR` dev deps, it would be nice to make sure there won't be any side effects since `MPL-2.0` was only captured in `DEV_ONLY_LICENSE_ALLOWED` before. @elastic/kibana-operations, can you please confirm?  

```
// The following list applies to packages both
// used as dependencies or dev dependencies
export const LICENSE_ALLOWED = [
```